src/urh/dev/native/lib/bladerf.pyx

```diff
@@ -129,8 +129,23 @@ cpdef int set_center_freq(bladerf_frequency frequency):
     return bladerf_set_frequency(_c_device, get_current_bladerf_channel(), frequency)
 
 cpdef bladerf_frequency get_center_freq():
-    cdef bladerf_frequency result = 0
-    err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+  cdef bladerf_frequency result = 0
+  err = bladerf_get_frequency(_c_device, get_current_bladerf_channel(), &result)
+
+  if err != 0:
+      return 0
+
+  return result
+
+cpdef int set_bias_tee(on_or_off):
+  IF BLADERF_API_VERSION >= 2:
```

Comment:
Can we add an ELSE case and return 0 or -1? I am not sure what Cython will do here for BLADERF_API_VERSIOn<2 when the method definition is empty (and implicitly returns None, which doesn't match the signature)